srfi-178.html

```diff
@@ -303,15 +306,13 @@ <h3>Selectors</h3>
 <h3>Iteration</h3>
 
 <p><code>(bitvector-take <i>bvec n</i>) -> bitvector</code> <br/>
-
-<p><code>(bitvector-take-right <i>bvec n</i>) -> bitvector</code> </p>
+<code>(bitvector-take-right <i>bvec n</i>) -> bitvector</code></p>
 
 <p>Returns a bitvector containing the first/last <i>n</i>
 elements of <i>bvec</i>.</p>
 
 <p><code>(bitvector-drop <i>bvec n</i>) -> bitvector</code> <br/>
-
-<p><code>(bitvector-drop-right <i>bvec n</i>) -> bitvector</code> </p>
+<code>(bitvector-drop-right <i>bvec n</i>) -> bitvector</code> </p>

```

Comment:
Did you want to remove the final space here, too?

Reply:
There are many many more occurrences of a trailing space after the </code> tag, and it doesn't pause a problem for my pattern matching converter, so I've left it there.